libparsec/crates/client/tests/unit/workspace/folder_transactions.rs

```diff
@@ -475,3 +475,244 @@ async fn ops_outbound_sync(ops: &WorkspaceOps) {
         }
     }
 }
+
+/// Test that renaming a file to match the prevent sync pattern make the file disappear from the parent in the remote
```

Comment:
```suggestion
/// Test that renaming a file or folder to match the prevent sync pattern make the file disappear from the parent in the remote
```

Reply:
Done.

FYI, For the filesystem POV a file or a folder are often refereed as a file, I used the same naming here